bindings/rabbitmq/rabbitmq.go

```diff
@@ -7,18 +7,26 @@ package rabbitmq
 
 import (
 	"encoding/json"
+	"strconv"
+	"time"
 
 	"github.com/dapr/components-contrib/bindings"
 	"github.com/dapr/dapr/pkg/logger"
 	"github.com/streadway/amqp"
 )
 
+const (
+	rabbitMQQueueMessageTTLKey = "x-message-ttl"
+)
+
 // RabbitMQ allows sending/receiving data to/from RabbitMQ
 type RabbitMQ struct {
-	connection *amqp.Connection
-	channel    *amqp.Channel
-	metadata   *rabbitMQMetadata
-	logger     logger.Logger
+	connection      *amqp.Connection
+	channel         *amqp.Channel
+	metadata        rabbitMQMetadata
+	logger          logger.Logger
+	queue           amqp.Queue
+	defaultQueueTTL *time.Duration
```

Comment:
I would expect ttl to be part of rabbitMQMetadata since it would be read from yaml.

Reply:
I moved the ttl to the metadata. I am not using json.Unmarshal to get it though. I am using the helper function to get it from a map[string]string. Is that ok?